JPF/utils.py

```diff
@@ -53,10 +53,10 @@ def mkdir_p(my_path):
 
 def connect():
 
-    user = os.environ['MYSQL_USER']
-    host = os.environ['MYSQL_HOST']
-    pw = os.environ['MYSQL_PASSWORD']
-    db = os.environ['MYSQL_DATABASE']
+    user = 'root'  # os.environ['MYSQL_USER']
+    host = 'localhost'  # os.environ['MYSQL_HOST']
+    pw = ''  # os.environ['MYSQL_PASSWORD']
+    db = 'heatseek'  # os.environ['MYSQL_DATABASE']
```

Comment:
Yeah, I just haven't bothered with it yet, I do have this auto-env shell extension but didn't bother setting it up. Also this is a purely local instance so leaking it doesn't matter.

Removed.